web/impact/impact/v1/classes/option_analysis.py

```diff
@@ -77,6 +83,7 @@ def analyses(self, option_spec):
             for option in options]
 
     def analysis(self, option_name, helper, spec_helper):
+        '''Produce an analysis for a particular option'''
```

Comment:
would be nice to detail what an analysis entails or the description of the returned datastructure here.